yarn.lock

```diff
@@ -2431,9 +2431,9 @@ __metadata:
     "@tanem/react-nprogress": ^4.0.8
     "@types/node": ^12.12.38
     "@types/rc-slider": ^8.6.6
-    "@types/react": ^16.14.23
+    "@types/react": ^16.14.0
     "@types/react-datepicker": ^3.1.8
-    "@types/react-dom": ^16.9.14
+    "@types/react-dom": ^16.9.7
```

Comment:
shouldn't be 16.14.0?

Reply:
https://github.com/logicalclocks/quartz/pull/215#discussion_r826102847